openedx/core/djangoapps/schedules/tasks.py

```diff
@@ -77,18 +81,60 @@ def _recurring_nudge_schedules_for_hour(target_hour, org_list, exclude_orgs=Fals
         course_id_str = str(enrollment.course_id)
         course = enrollment.course
 
-        course_root = reverse('course_root', args=[course_id_str])
-
-        def absolute_url(relative_path):
-            return u'{}{}'.format(settings.LMS_ROOT_URL, urlquote(relative_path))
+        course_root_relative_url = reverse('course_root', args=[course_id_str])
+        dashboard_relative_url = reverse('dashboard')
 
         template_context = {
             'student_name': user.profile.name,
             'course_name': course.display_name,
-            'course_url': absolute_url(course_root),
+            'course_url': absolute_url(course_root_relative_url),
+
+            # Platform information
+            'homepage_url': encode_url(marketing_link('ROOT')),
+            'dashboard_url': absolute_url(dashboard_relative_url),
+            'template_revision': get_scm_revision(),
+            'template_tag': get_scm_tag(),
+            'platform_name': settings.PLATFORM_NAME,
+            'contact_mailing_address': settings.CONTACT_MAILING_ADDRESS,
+            'social_media_urls': encode_urls_in_dict(getattr(settings, 'SOCIAL_MEDIA_FOOTER_URLS', {})),
+            'mobile_store_urls': encode_urls_in_dict(getattr(settings, 'MOBILE_STORE_URLS', {})),
 
             # This is used by the bulk email optout policy
             'course_id': course_id_str,
         }
 
         yield (user, course.language, template_context)
+
+
+def get_scm_revision():
+    try:
+        return check_output(['git', 'log', '-1', '--format=%H']).strip()
+    except CalledProcessError:
+        return ''
+
+
+def get_scm_tag():
+    try:
+        return check_output(['git', 'describe', '--always', '--tags']).strip()
+    except CalledProcessError:
+        return ''
+
+
+def encode_url(url):
+    # Sailthru has a bug where URLs that contain "+" characters in their path components are misinterpreted
+    # when GA instrumentation is enabled. We need to percent-encode the path segments of all URLs that are
+    # injected into our templates to work around this issue.
+    parsed_url = urlparse(url)
+    modified_url = parsed_url._replace(path=urlquote(parsed_url.path))
+    return modified_url.geturl()
+
+
+def absolute_url(relative_path):
+    return encode_url(u'{}{}'.format(settings.LMS_ROOT_URL, relative_path))
```

Comment:
You may run into issues with slashes `'/'`. Make sure the setting and parameter always meet expectations:

```python
root = settings.LMS_URL_ROOT.rstrip('/')
relative_path = relative_path.lstrip('/')
return encode_url(u'{root}/{path}'.format(root=root, path=relative_path))
```

Reply:
Good idea, done.